rskj-core/src/test/java/co/rsk/rpc/netty/Web3WebSocketServerTest.java

```diff
@@ -68,13 +70,24 @@ public void smokeTest() throws Exception {
         String mockResult = "output";
         when(web3Mock.web3_sha3(anyString())).thenReturn(mockResult);
 
-        int randomPort = 9998;//new ServerSocket(0).getLocalPort();
+        int randomPort = 9998;
+
```

Comment:
maybe I'd add another test in which we use a short timeout (like 100 ms), then use a mock that sleeps longer than that, and assert that the exception we expect is thrown, and  / or that the connection is closed altogether

Reply:
well I thought this one to be tackled in the integration tests, but it'd be nice to have a very simple one here

Reply:
we'll move forward and add this one in a [separate issue](https://github.com/rsksmart/rskj/issues/1590)

Reply:
nice